.github/workflows/update-repo-request-template.yml

```diff
@@ -0,0 +1,82 @@
+  push:
+    paths:
+      - '.github/org/**'
+      - '.github/workflows/update-repo-request-template.yml'
+
+permissions:
+  contents: write
+
+env:
+  ORG_FILE: .github/org/org-structure.yml
+  ISSUE_TEMPLATE_FILE: .github/ISSUE_TEMPLATE/repo-request.yml
+
+jobs:
+  update-org:
+    runs-on: ubuntu-latest    
+    steps:
+      - uses: actions/checkout@v3.3.0
+      - name: Get Companies
+        shell: bash
+        run: >-
+          find .github/org/companies -mindepth 1 -maxdepth 1 -type d -exec sh -c 'echo - $(basename "$1") \($(yq '.number' "$1"/metadata.yml)\)' sh {} \; | sort > companies
+      - name: Get Teams
+        shell: bash
+        run: >-
+          find .github/org/companies/*/teams -mindepth 1 -maxdepth 1 -type d -exec sh -c 'echo - $(yq '.id' "$1"/metadata.yml)-$(basename "$1")' sh {} \; | sort > teams
+      - name: Get Projects
+        shell: bash
+        run: >-
+          find .github/org/companies/*/teams/*/projects -mindepth 1 -maxdepth 1 -type d -exec sh -c 'echo - $(basename "$1")' sh {} \; | sort > projects
+      - name: Update Companies
+        shell: bash
+        run: >-
+          COMPANIES=$(cat companies)
+          yq -i '(.body[] | select(.type=="dropdown" and .attributes.label=="Company") | .attributes.options) |=env(COMPANIES)'
+          $ISSUE_TEMPLATE_FILE
+      - name: Update Teams
+        shell: bash
+        run: >-
+          TEAMS=$(cat teams)
+          yq -i '(.body[] | select(.type=="dropdown" and .attributes.label=="Team") | .attributes.options) |=env(TEAMS)'
+          $ISSUE_TEMPLATE_FILE
+      - name: Update Projects
+        shell: bash
+        run: >-
+          PROJECTS=$(cat projects)
+          yq -i '(.body[] | select(.type=="dropdown" and .attributes.label=="Project") | .attributes.options) |=env(PROJECTS)'
+          $ISSUE_TEMPLATE_FILE
+      - name: Determine Changes
+        id: change-detection
+        shell: bash
+        run: |
+          if (git diff --quiet .github/ISSUE_TEMPLATE/repo-request.yml)
+          then 
+            echo changed=false >> "$GITHUB_OUTPUT"
+          else 
+            echo changed=true >> "$GITHUB_OUTPUT"
+          fi
+      - name: Display File
+        if: ${{ steps.change-detection.outputs.changed == 'true' }}
+        shell: bash
+        run: cat $ISSUE_TEMPLATE_FILE
+      - name: 'Yamllint'
+        if: ${{ steps.change-detection.outputs.changed == 'true' }}
+        uses: karancode/yamllint-github-action@master
+        with:
+          yamllint_file_or_dir: ${{ env.ISSUE_TEMPLATE_FILE }}
+          yamllint_strict: true
+      - name: Commit Changes          
+        if: ${{ steps.change-detection.outputs.changed == 'true' }}
+        env: 
+          CI_COMMIT_MESSAGE: "chore(org): Update Issue Template File"
+          CI_COMMIT_AUTHOR: Continuous Integration
+        run: |
+          git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
+          git config --global user.email "CI@users.noreply.github.com"
```

Comment:
Can this be achieved using a GitHub App? We have an app available in the xpirit-training repo that we can use for that.